tests/lib/rules/prop-types.js

```diff
@@ -2445,7 +2443,6 @@ ruleTester.run('prop-types', rule, {
       parser: 'babel-eslint',
       errors: [
         {message: '\'names\' is missing in props validation'},
-        {message: '\'names.map\' is missing in props validation'},
```

Comment:
good fix here

Reply:
@ljharb Am I good on all of the changes you asked for?